code/modules/events/anomaly/anomaly_ectoplasm.dm

```diff
@@ -0,0 +1,74 @@
+#define ANOMALY_INTENSITY_MINOR "anomaly_intensity_minor"
+#define ANOMALY_INTENSITY_MODERATE "anomaly_intensity_moderate"
+#define ANOMALY_INTENSITY_MAJOR "anomaly_intensity_major"
```

Comment:
These are user-facing when you pass it into tgui_input_list, so make it something actually readable like "Minor Intensity".